src/AndcultureCode.CSharp.Sitefinity.Core/Models/Services/AuthenticateResponse.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Newtonsoft.Json;
+
+namespace AndcultureCode.CSharp.Sitefinity.Core.Models.Services
+{
+    public class AuthenticateResponse
+    {
+        [JsonProperty("access_token")]
+        public string AccessToken { get; set; }
+        /// <summary>
+        /// The number of seconds will Sitefinity expires the session.
```

Comment:
Consider changing the language to `The number of seconds before Sitefinity will expire the session`